ospd/ospd.py

```diff
@@ -1423,6 +1423,9 @@ def handle_command(self, command: str) -> str:
         if not command and tree.tag != "authenticate":
             raise OspdCommandError('Bogus command name')
 
+        if tree.tag == "get_vts":
```

Comment:
I don't think we should re-add this. We already know the kind of command and all commands should handle requests the same.